packages/webui/src/client/ui/RundownView/SelectedElementsContext.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import {
+	AdLibActionId,
+	PartInstanceId,
+	PieceId,
+	RundownId,
+	SegmentId,
+} from '@sofie-automation/corelib/dist/dataModel/Ids'
+
+interface RundownElement {
+	type: 'rundown'
+	id: string
```

Comment:
While what `elementId` is an id _of_ is clear from the typings, what this `id` is representing is more opaque and I think it would be nice if there was some sort of a definiton of what this id represents, especially since it's the API user's responsibility to create this ID string. It would be good to explain how to do that, since non-unique IDs here will cause all sorts of havoc.

Reply:
Good point. I've removed the id. And made a union type for elementId, so this will be the reference.
type ElementId = SelectedElement['elementId']